aspnetcore/security/authentication/certauth.md

```diff
@@ -233,8 +233,17 @@ See the [host and deploy documentation](xref:host-and-deploy/proxy-load-balancer
 
 No forwarding configuration is required for Azure. This is already setup in the certificate forwarding middleware.
 
+Azure Application Service sends certificates in the `X-ARR-ClientCert` header. To configure the middleware for Azure Application Services, use the following code:
+ 
+ ```csharp
+  services.AddCertificateForwarding(options =>
+  	{
+ 		options.CertificateHeader = "X-ARR-ClientCert";
+ 	}
+ ```
+
```

Comment:
Azure Application Service sends certificates in the `X-ARR-ClientCert` header. To configure the middleware for Azure Application Services, use the following code:

 ```csharp
  services.AddCertificateForwarding(options =>
  	{
 		options.CertificateHeader = "X-ARR-ClientCert";
 	}
 ```